action.yml

```diff
@@ -26,6 +26,10 @@ inputs:
     description: 'Error out when we have an invalid resource. Default: false'
     required: false
     default: 'false'
+  token:
+    description: 'Used to pull release info from GitHub. Does not need to be supplied on github.com.'
+    required: false
+    default: ${{ github.server_url == 'https://github.com' && github.token || '' }}
```

Comment:
Why not just:
```suggestion
    default: ${{ github.token }}
```

Reply:
Because then it would be set on GHE instances which would cause the action to fail instead of making the request anonymously.

Reply:
Similar usage to [actions/setup-python](actions/setup-python)